distribution/ddf-common/src/main/resources/bin/ddf.bat

```diff
@@ -4,24 +4,58 @@ SETLOCAL
 SET ARGS=%*
 SET DIRNAME=%~dp0%
 
+PUSHD %DIRNAME%\..
+SET DDF_HOME=%CD%
+POPD
+
```

Comment:
This is the Windows script. In the default configuration, it manages starting and stopping Solr.